pkg/walleterror/walleterror.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright Avast Software. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Package walleterror define error model for go api.
```

Comment:
Should say `Package walleterror defines the error model for the Go api.`

Reply:
done

Reply:
done